tensorflow/lite/toco/tooling_util.cc

```diff
@@ -2327,7 +2327,7 @@ void UseArraysExtraInfo(Model* model, bool quantize_output) {
         // Make sure to create the shape even if there are no dims, to
         // correctly record 0-D shapes.
         array.mutable_shape();
-        for (int dim : entry.shape().dims()) {
+        for (const auto & dim : entry.shape().dims()) {
```

Comment:
nit: `const auto&`

Reply:
thanks @srjoglekar246 , i have updated the code as per your comments, kindly check and approve.